internal/pkg/model/object.go

```diff
@@ -110,7 +107,7 @@ type Config struct {
 	Description       string                 `json:"description" diff:"true" descriptionFile:"true"`
 	ChangeDescription string                 `json:"changeDescription"`
 	Content           *orderedmap.OrderedMap `json:"configuration" validate:"required" diff:"true" configFile:"true"`
-	Blocks            Blocks                 `json:"-" validate:"dive" diff:"true"`
+	Transformation    *Transformation        `json:"-" validate:"omitempty,dive" diff:"true"`
 	Orchestration     *Orchestration         `json:"-" validate:"omitempty,dive" diff:"true"`
 	Relations         Relations              `json:"-" validate:"dive" diff:"true"`
```

Comment:
`Blocks` -> `Transformation.Blocks`:
- zjednotene s `Orchestration` vid nizsie
- potrebujem lepsie reprezentovat `shared code` a toto je krok k tomu
- do transformation mozem nasledne ulozit `shared_code_id` a `shared_code_row_ids`.
- a ked tam to ID budem mat, tak uz sa automaticky nahradi v sablone